README.md

```diff
@@ -140,6 +140,7 @@ The component accepts the following props:
 |     **`content`**     | function | A function that returns a component reference value. The content of this reference value is then used for print                     |
 |   **`copyStyles`**    | boolean  | Copies all &lt;style> and &lt;link type="stylesheet" /> from <head> inside the parent window into the print window. (default: true) |
 |  **`onBeforePrint`**  | function | Optional callback function that triggers before print. Either returns void or a Promise. If the function returns a Promise the content will be printed when the Promise is resolved. Users are responsible for catching the Promise rejecting.                                                                                     |
```

Comment:
Let's update this to:

Optional callback function that triggers before print. Either returns void or a Promise. If the function returns a Promise the content will be printed when the Promise is resolved. Users are responsible for catching the Promise rejecting. Note: this function is run immediately prior to printing, but after the page's content has been gathered. To modify content before printing, use `onBeforeGetContent` instead.

Reply:
ok I'´ll do it now.